### PR TITLE
Changes on display

### PR DESCRIPTION
**Pull request by XinYenFon**
Signed-off-by: Antes antes@simplemachines.org

### CODE REVIEW

File: Themes/default/Display.template.php [hunk 1]
```diff
@@ -23,9 +23,32 @@ function template_main()
 			</div>';
 	}
 
-	// Show the anchor for the top and for the first message. If the first message is new, say so.
+	// Show new topic info here?
 	echo '
+		<div id="display_head">
+			<h2 class="display_title">'. $context['subject']. '', ($context['is_locked']) ? '<span class="generic_icons lock"></span>' : '','', ($context['is_sticky']) ? '<span class="generic_icons sticky"></span>' : '','</h2>
+			<p>',$txt['started_by'],' ', $context['topic_poster_name'],', ', $context['topic_started_time'],'</p>';
```
Comment (live627) [inline Themes/default/Display.template.php:30]: This seems a little messy to me...

I'd change it to:
`<h2 class="display_title">', $context['subject'], ($context['is_locked']) ? '<span class="generic_icons lock"></span>' : '', ($context['is_sticky']) ? '<span class="generic_icons sticky"></span>' : '', '</h2>`

**Comment by Oldiesmann** [comment]
Two things...
1. You removed the code to display the poll question.
2. Why are we removing those icons? Similar icons (using generic_icons rather than topic-icon) are shown on the messageindex.

**Comment by XinYenFon** [comment]
1) I'm just showing Question as title (beside static Poll)
2) yes that's why, consistency... and using less stuff.

**Comment by Oldiesmann** [comment]
Ok. Didn't see either of those :tongue: 

I still think the poll question looks fine where it is though. Don't see any reason to put it up so far above the poll options.

**Comment by Oldiesmann** [comment]
Looking at this again, I just realized something. Only 2 of the 8 icons from the topic icons sprite that are currently used are also in generic icons. However, with a slight tweak to the determineTopicClass function we could also use another 8 of them ("my" topic/poll vs a topic/poll someone else started), so we should probably combine those as well.

The rest are for "hot" and "very hot" topics, which is no longer a feature in SMF.

**Comment by XinYenFon** [comment]
I did this because smf does not allow second questions also removed margin and some padding they look more close.

-----Original Message-----

**Pull request closed without merging**